ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,77 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+releases:
```

Comment:
From the following item, the upgrade path would be like 4.11 -> 4.10 -> 4.11, looks like something wrong.

Reply:
I've changed base images to 4.11, I think it was upgrading to latest stable 4.10 -> latest stable 4.11 -> latest nightly 4.11  based on line #57 . I'll run the rehearse job again.

Reply:
I referred openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-eus-4.10.yaml, per your requirement, seem like need to define `initial` and `latest` to stable 4.10, `intermediate` to stable 4.11, and `target` to nightly 4.11.

Reply:
If we use 
```
commands: echo "${RELEASE_IMAGE_INTERMEDIATE410},${RELEASE_IMAGE_STABLE},${RELEASE_IMAGE_TARGET}"
        > ${SHARED_DIR}/upgrade-edge
```
the upgrade path is gonna be RELEASE_IMAGE_LATEST -> RELEASE_IMAGE_INTERMEDIATE410 -> RELEASE_IMAGE_STABLE -> RELEASE_IMAGE_TARGET.

What's your expected upgrade path?